src/modules/wlr/taskbar.cpp

```diff
@@ -661,9 +661,10 @@ void Taskbar::register_manager(struct wl_registry *registry, uint32_t name, uint
         spdlog::warn("Register foreign toplevel manager again although already existing!");
         return;
     }
-    if (version < ZWLR_FOREIGN_TOPLEVEL_HANDLE_V1_STATE_FULLSCREEN_SINCE_VERSION) {
-        spdlog::warn("Using different foreign toplevel manager protocol version: {}", version);
+    if (version <= ZWLR_FOREIGN_TOPLEVEL_HANDLE_V1_STATE_FULLSCREEN_SINCE_VERSION) {
```

Comment:
`version <= 2`? Is there a reason to require version 3?
Also, strictly speaking, the module should work just fine with version 1 and 2 is not required. wayland protocols are backward compatible and supporting version X means that everything  < X should work as well.

Reply:
I require at least version 2, because the taskbar uses the fullscreen functionality of the foreign-toplevel-manager. Hence, that's why I require at least version 2 ... Does this makes sense?

Reply:
Your change in condition now requires 3 and prints warning for 2.
That does make sense, but as I said, taskbar would work without fullscreen functionality. Although we don't have compositors with version 1 to test that :smile:

Reply:
Ah sorry. You are right. I got the condition totally wrong in my head. I will update the warning as well.

Reply:
Oh, right, there are methods with since="2"...
Passive usage (like checking flags) does not require anything. Calling `set_fullscreen` may cause a protocol error and it would be better to have a check in `Task::fullscreen`.  Something like
```c++
if (zwlr_foreign_toplevel_handle_v1_get_version(handle_) < ZWLR_FOREIGN_TOPLEVEL_HANDLE_V1_STATE_FULLSCREEN_SINCE_VERSION) {
    fmt::warn(...);
    return;
}
```

Reply:
Fixed in the latest version. Thank you very much for the feedback.